api/backend_test.go

```diff
@@ -1072,7 +1121,7 @@ func login(t *testing.T, conf *params.NodeConfig) {
 func TestLoginAndMigrationsStillWorkWithExistingUsers(t *testing.T) {
 	utils.Init()
 
-	srcFolder := "../static/test-0.97.3-account/"
+	srcFolder := "../static/test-0.132.0-account/"
```

Comment:
Yes, we cannot use that old Status folder any more.

In that version `0.97.3` key_uid was not a part of `accounts_table` and we're unable to do a migration properly with new migration sql script added to this PR. There is no way to get a `key_uid` from anywhere for already added accounts.

Reply:
does it mean we introduced a break change here ?

Reply:
Well, is it a break change or not... I would say it is not, since it doesn't break anything. It refers to locally stored Status folder which is used for tests. At that point when we were using `0.97.3` version, `key_uid` column was not part of `accounts` table as well as many other columns. Check the `accounts` table at the location `/status-go/static/test-0.97.3-account/0x7c46c8f6f059ab72d524f2a6d356904db30bb0392636172ab3929a6bd2220f84.db` and you will see how it looked at that time. Was no chance to do a proper migration without having `key_uid` for each account.

Reply:
sorry, when i said it's a break change, i mean it would be a break change for earlier mobile user?

Reply:
> sorry, when i said it's a break change, i mean it would be a break change for earlier mobile user?

Yes, that's true. So if user has a very old local `Status` folder and hasn't run the app since that version (which was present in May 2022) this will break. Though that should be very unlikely, but even if there are such users we shouldn't worry too much about them we're still under the development, that will be more dangerous after mvp.